app/models/icon.rb

```diff
@@ -4,5 +4,4 @@ class Icon < ApplicationRecord
   default_scope -> { kept }
 
   belongs_to :portfolio_item
-  validates :data, :presence => true
```

Comment:
When is this the case on topology?  Seems like we would not want to create an empty icon record.

Reply:
Yes, it isn't required on the model, https://github.com/ManageIQ/topological_inventory-core/blob/master/app/models/service_offering_icon.rb